#!/usr/bin/env python
PACKAGE = "map_follower"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cmd_hz", double_t, 0, "cmd_hz value",    50.0, 50.0,   200.0)
gen.add("desired_distance", double_t, 0, "desired_distance value",    1.5, 0.0,   3.0)
gen.add("lookahead", double_t, 0, "lookahead",    2.0, 0.0,   50.0)
gen.add("kp", double_t, 0, "kp",    1.0, 0.0,   100.0)
gen.add("kd", double_t, 0, "kp",    0.0, 0.0,   1.0)
gen.add("ki", double_t, 0, "kp",    1e-3, 0.0,   1.0)
gen.add("speed_scale", double_t, 0, "speed_scale",    0.93, 0.0,   3.0)
gen.add("angle_scale", double_t, 0, "angle_scale",    0.75, 0.0,   3.0)




exit(gen.generate(PACKAGE, "map_follower", "dyn_params"))